backend/src/services/v2/release.ts

```diff
@@ -144,6 +144,35 @@ export async function updateRelease(user: UserDoc, modelId: string, semver: stri
   return updatedRelease
 }
 
+export async function newReleaseComment(user: UserDoc, modelId: string, semver: string, comment: string) {
+  const model = await getModelById(user, modelId)
+  const release = await getReleaseBySemver(user, modelId, semver)
+
+  release.comments.push({
+    comment,
+    user: user.dn,
+    createdAt: new Date().toISOString(),
+  })
+
+  await validateRelease(user, model, release)
+
+  const auth = await authorisation.release(user, model, release, ReleaseAction.View)
```

Comment:
You don't need to check if we can view the release, we've already checked this when we fetch it!